tests/lib/gcf_sms_lib/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  gcf_sms_lib.functionality_test:
+    platform_allow: nrf9160dk_nrf9160_ns
```

Comment:
Please add integration platform option to make it possible to build this test in Jenkins CI:
```
    integration_platforms:
      - nrf9160dk_nrf9160_ns
```